openpaas-james/mailbox/opendistro-tests/src/test/java/org/apache/james/mailbox/opendistro/DockerAuthOpenDistroSingleton.java

```diff
@@ -0,0 +1,28 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
```

Comment:
why apache license?

Reply:
I'm not sure. I thought stuffs copied from apache/james project should have their license?

Reply:
Different name, very small class, generic enough... I'm not sure we can say it's 'copied'. But I'm honestly not sure on this one, so if you prefer to let it there I'm fine too